### PR TITLE
Setup Basic SearchPage

### PR DESCRIPTION
**Pull request by Sina-Sche**
- [x] Finish components for SearchPage

- [x] Edit styling of components

- [x]  Setup SearchPage (still hard-coded)

### CODE REVIEW

**Comment by kschmitz123** [comment]
🎉

File: client/src/components/ProductText.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components/macro";
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 1rem;
```
Comment (ncbecker) [inline client/src/components/ProductText.js:6]: Sometimes you use rem and sometimes px as unit for margin and padding. If possible I would choose one for more consistency.

**Review by ncbecker — APPROVED** [review]
Looks good 👍

File: client/src/components/ProductText.js (hunk 1 above)
Comment (Sina-Sche) [reply client/src/components/ProductText.js:6]: Thanks! You're right! I changed my mind about using px/rem a few times. Will spend some time this week refactoring the code to make it more consistent!

**Pull request merged**